src/org-manager/components/organisation-details-info/organisation-details-info.component.html

```diff
@@ -1,5 +1,15 @@
 <ng-container *ngIf="org; else noOrg">
-  <h1 class="govuk-heading-xl">Organisation details</h1>
+    <ng-container *ngIf="org.status=='PENDING'; else showheading">
+      <h1 class="govuk-heading-xl">Registration details</h1>
+      <h2 class="govuk-heading-l">Organisation details</h2>
+      <strong style="display: inline-block" class="govuk-heading-s">Registration status</strong>
+      <span style="margin-left: 10px" class="hmcts-badge">
+        {{org.status}}
```

Comment:
If you've checked above that this is `'PENDING'`, you don't need a second binding here...

Reply:
you mean I need to put 'PENDING' instead of binding?

Reply:
removed binding and put 'PENDING' text